tasks/section4.yml

```diff
@@ -15,12 +15,13 @@
   when: ubu16gsa_auditd_config == true
   tags:
       - id_4.1.1.3
-
+#FIXME Added ignoreerrors due to Auditd not fully compatiable inside containers
 - name: "4.1.2 | Ensure auditd service is enabled"
   systemd:
       name: auditd
       state: started
       enabled: yes
+  ignore_errors: yes
```

Comment:
`ignore_errors` probably isn't what you want, since this could cause errors to go unnoticed when running the playbook. If you want to exclude a task in test, you should probably tag it with something like `notest` and then use `--skip-tags notest` in testing.

Reply:
If this is specific to systemd in docker, you're probably missing some important settings. At minimum, I found that you need to provide the container with the SYS_ADMIN capability and mount `/sys/fs/cgroup` into the container. [Molecule docs](https://molecule.readthedocs.io/en/latest/examples.html#systemd-container) have some additional examples.